.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,20 @@
+// For format details, see https://aka.ms/devcontainer.json.
```

Comment:
Note: Tested creation of local container registry, kind clusters, and deployment of serving and ingress with [this setup script](https://gist.github.com/samruddhikhandale/e93358e7cf3a222e7624f70960098a14) in a codespace. I can add it to the `postCreateCommand` if necessary.

Reply:
Added!